text/safrole.tex

```diff
@@ -254,7 +254,7 @@ \subsection{The Extrinsic and Tickets}
   \end{cases}
 \end{align}
 
-We define $\mathbf{n}$ as the set of new tickets, with the ticket identity, a hash, defined as the output component of the Bandersnatch Ring\textsc{vrf} proof:
+We define $\mathbf{n}$ as the set of new tickets, with the ticket identifier, a hash, defined as the output component of the Bandersnatch Ring\textsc{vrf} proof:
```

Comment:
I find "identifier" less ambiguous. To not be confused with the "identity" function. But this is very nitpicky :-)